mark48-sp

marvalkrystof opened the pull request:
Pilotní verze semestrální práce dostupná z https://eso.vse.cz/~mark48/sp/

marvalkrystof committed:
mark48-sp

nvbach91 commented:
jak se prosím dostanu do administrační části?

marvalkrystof commented:
Přihlašte se pod adminem.
admin@example.com
password123

nvbach91 reviewed (changes requested):
- některé navigační odkazy obsahují lomítko navíc
- proč se zobrazuje [událost s ID 7 dvakrát](https://github.com/user-attachments/assets/b7a3dcba-4b47-4081-a3ea-d9910d0f9cc9) v `admin/events.php`
- přidat počet vstupenek do Recent orders v Dashboardu
- po vygenerování seating planu už nejde přidat další místa?
- během editace Seating plan: [failed to release lock](https://github.com/user-attachments/assets/bc05d0e8-eaa7-415e-b7e6-b207e4d580f6) a už nejde nic udělat, ani log out

marvalkrystof committed:
mark48-sp-fixes

marvalkrystof commented:
1. Fixed
2. Fixed, místo poslední události se renderovala 2x předposlední
3. Přidáno
4. Nyní lze měnit dimenze seating planu (jen pokud event nemá prodanou žádnou vstupenku)
5. Fixed

nvbach91 commented:
- obrazky?
- jak funguje 15 min
- jak pocet ticketu
- past events, upcoming events
- seat category separate for each event, or use as margin
=====
- zvyrazneni menu
- obsah qr kodu/ API- https://api.gokasa.eu/public/qrcode?text=975d4333-f1a0-4cc4-971a-45794d4dfc1a
- 6 znaku => 8 znaku + slozitost

marvalkrystof committed:
mark48-sp-obhajoba-fixes
